ext/com_dotnet/tests/bug45280.phpt

```diff
@@ -8,8 +8,9 @@ if (!extension_loaded("com_dotnet")){ echo "skip COM/.Net support not present";
 <?php
 $dict = new COM("Scripting.Dictionary");
 
+$reflection = new ReflectionObject($dict);
 ob_start();
-ReflectionObject::export($dict);
+(string) $reflection;
```

Comment:
Should be `echo $reflection;` probably?

Reply:
Oh,  why I didn't think about it... :) I'll change it.